Advanced-Recruit-Training/README.adoc

```diff
@@ -0,0 +1,52 @@
+= U of C Solar Car Advanced Recruit Training
+Intro project to learn c++ and QT
+
+== Running the project
+
+=== Via Command Line
+
+To build this project on linux navigate to the source directory and run the following commands:
+
+`qmake`
+
+`make`
+
+`./batteryLife` - The command line arguments are as follows:
+
+[options="header"]
+|============================
+|Flag|Argument |Description
+|`-f` or `--filename` |`<filename>`|Name of csv log file of battery data
+
+|`-i` or  `--initStateOfCharge` |`<stateOfCharge>`|Initial State Of Charge of Battery as percent from 0-100
+|============================
+
+For example, to run the project opening `myData.csv` and setting the initState of charge as 50%: 
+
+`./batteryLife -f myData.csv -i 50`
+
+=== Via QT Creator
+In QT Creator, the above `qmake` and `make` are located in `Build`, and the project can be run through the play button. 
+You'll have to set the appropriate command line arguments in `Projects > Build & Run > Run`. 
+
+== Objective
+
+The goal of this project is to give new recuits a good starting point to learn the required skills to work on the other projects. 
+If you ever need any help with how to use Qt's functions, a good start would be to look at http://doc.qt.io/qt-5.6/reference-overview.html[Qt's reference manual]
+
+=== Tasks to Complete
+
+ *  Complete the error checking in `LogFileReader::parseLine`. 
+ The proper format of each line in batData.csv should be hh:mm:ss:zzz, so any other differences should be caught. 
+ It should be able to catch all the errors in batDataFailure.csv.
+ *  Implement the functions of BatteryStateOfChargeService. 
+ See the notes below on how to calculate the values.
+ *  Print out the time until depleted or charged (found in BatteryStateOfChargeService) in `BatteryStateDisplayService::handleBatteryDataReceived`.
+
+=== Values to Note for BatteryStateOfChargeService
+
+ *  The battery is charging when the battery current is negative, depleting when it is positive.
+ *  The initial amount of Amphours used would be the Battery's Amphour capacity(Ah) x it's state of charge(%).
```

Comment:
Instead of using x i would say multiplied, consistency with below and since this is still a sentence I think it fits better.